packages/driver/cypress/fixtures/dom.html

```diff
@@ -437,8 +437,8 @@
       </select>
 
       <select name="foods">
-        <option value="Japanese">Ramen</option>
-        <option value="Ramen">Japanese</option>
+        <option value="Ramen">Ramen</option>
```

Comment:
Why the reorder?

Reply:
I figured the `option` with the value `Ramen` probably should have the text `Ramen` and same with `Japanese`. Seemed like a typo so I went ahead and made the values match and all tests seem to be in working order.

Reply:
lol, I completely missed that!

Reply:
Haha that's a good catch 😆